### PR TITLE
Update basic & blink handouts

### PR DESCRIPTION
**Pull request by encukou**
- Use sections
- Use friendlier wording for other distros
- Make the important commands more prominent
- Wording changes, typo fixes

### CODE REVIEW

**Review by hroncok — APPROVED** [review]

**Comment by encukou** [comment]
Here's one for blink as well.

File: blink/handout.tex [hunk 1]
```diff
@@ -39,13 +39,6 @@
 
 \setlist[enumerate,1]{start=0}
 
-\newcommand\plpage{
-    \newpage
-    \begin{tikzpicture}[remember picture,overlay]
-        \node[opacity=0.1,below left,xshift=1cm,yshift=1cm] at (current page.north east) {\includegraphics[width=12cm]{../logo}};
```
Comment (hroncok) [inline blink/handout.tex:45]: I guess we can now delete the logo entirely.

File: blink/handout.tex [hunk 2]
```diff
@@ -66,79 +59,96 @@
...
+      Create an input Pin for it:
 
       \texttt{~~~~button = Pin(0, Pin.IN)}
 
 \item Read the value with \texttt{button.value()}. What is it?
 
 \item Read the value while holding the \emph{BOOT} button. What is it?
 
+    \textit{
+        The \emph{BOOT} button is next to the USB connector.
```
Comment (hroncok) [inline blink/handout.tex:113]:
```suggestion
        The \emph{BOOT} button is also next to the USB connector.
```

File: blink/handout.tex [hunk 3]
```diff
@@ -66,79 +59,96 @@
...
 
-\item Set 0 to turn the light off: \texttt{led.value(0)}
+      \texttt{led = Pin(2, Pin.OUT)}
+
+\item Write 1 or 0 to turn the light on or off:
+
+    \texttt{
+        led.value(1) \\
+        led.value(0)
+    }
```
Comment (hroncok) [inline blink/handout.tex:94]: Not a blocker for now, but we should do this differently. Perhaps with minted.

File: blink/handout.tex [hunk 4]
```diff
@@ -165,17 +176,20 @@ \section*{Blinking LED with MicroPython on ESP32}
...
+      1023 means always.
       When experimenting, mind the fact that humans don't perceive light intensity linearly.
 
-      With PWM running, you can do other things in Python -- it is not blocking.
+      With a PWM running, you can do other things in Python -- it is not blocking.
       To change a PWM's frequency or duty, use the \texttt{pwm.freq(...)} or \texttt{pwm.duty(...)} methods.
-      To stop a PWM, use the \texttt{pwm.deinit()} method.
+      To stop a PWM, use \texttt{pwm.deinit()}.
+
+      Try it!
```
Comment (hroncok) [inline blink/handout.tex]:
```suggestion
      Try it out!
```

File: blink/handout.tex (hunk 1 above)
Comment (encukou) [reply blink/handout.tex:45]: AFAICS, it's not in the repo in the first place

File: blink/handout.tex (hunk 3 above)
Comment (encukou) [reply blink/handout.tex:94]: yup, leave this for later

File: blink/handout.tex (hunk 2 above)
Comment (encukou) [reply blink/handout.tex:113]: The previous mention of EN is on the previous handout, so I wouldn't use also here.

File: blink/handout.tex (hunk 1 above)
Comment (hroncok) [reply blink/handout.tex:45]: heh, it's ignored because it's pdf ;)

**Review by hroncok — APPROVED** [review]

**Pull request merged**